### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
MS style guide, make it easier for non native speakers who use MT (machine translation, ie, automatic browser translation)

### CODE REVIEW

**Comment by mthalman** [comment]
Thanks @Rick-Anderson. The READMEs in this repo are dynamically generated from templates. Can you apply these changes to the related template files and then regenerate the READMEs? Instructions are here: https://github.com/dotnet/dotnet-docker/blob/main/CONTRIBUTING.md#editing-readmes.

The related templates for the changes you've made are:
* https://github.com/dotnet/dotnet-docker/blob/main/eng/readme-templates/About.product-family.md
* https://github.com/dotnet/dotnet-docker/blob/main/eng/readme-templates/Use.product-family.md
* https://github.com/dotnet/dotnet-docker/blob/main/eng/readme-templates/RelatedRepos.md

File: README.md [hunk 1]
```diff
@@ -63,6 +66,7 @@ See [Hosting ASP.NET Core Images with Docker over HTTPS](https://github.com/dotn
 * [dotnet/nightly/runtime](https://hub.docker.com/_/microsoft-dotnet-nightly-runtime/): .NET Runtime (Preview)
 * [dotnet/nightly/runtime-deps](https://hub.docker.com/_/microsoft-dotnet-nightly-runtime-deps/): .NET Runtime Dependencies (Preview)
 * [dotnet/nightly/monitor](https://hub.docker.com/_/microsoft-dotnet-nightly-monitor/): .NET Monitor Tool (Preview)
+* [tnet/sdk-container-builds](https://github.com/dotnet/sdk-container-builds): .NET SDK Container Building Tools
```
Comment (MichaelSimons) [inline README.md:69]: This doesn't feel appropriate in this context.  The rest of these are all related docker repos.  The primary audience of this readme is via docker-hub [here](https://hub.docker.com/_/microsoft-dotnet/).

Comment (Rick-Anderson) [reply README.md:69]: @MichaelSimons your assertion is that .NET SDK Container Building Tools has no value to for https://github.com/dotnet/dotnet-docker customers and is not a Related .NET repo per the heading?

**Comment by Rick-Anderson** [comment]
Moved to #4153

File: README.md (hunk 1 above)
Comment (MichaelSimons) [reply README.md:69]: I am not necessarily saying that the link doesn't belong in the readme.  I am saying it doesn't belong in this section.  This section is strictly for related Docker repos.

Comment (Rick-Anderson) [reply README.md:69]: As a non-expert, I don't see the strict requirement
image

Comment (MichaelSimons) [reply README.md:69]: The sections of the readme follow the [Microsoft Artifact Registry Documentation Guidelines](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/containers-bburns/microsoft-container-registry/microsoft-artifact-registry-also-known-as-microsoft-container-registry-docs/onboardingdocs/documentationstandards).  The related repos section is intended to list related MCR/MAR repos not related github repos.

Comment (Rick-Anderson) [reply README.md:69]: @MichaelSimons can you PR this and put it in the right place?

Comment (Rick-Anderson) [reply README.md:69]: This PR was addressing [Microsoft Artifact Registry Documentation Guidelines](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/containers-bburns/microsoft-container-registry/microsoft-artifact-registry-also-known-as-microsoft-container-registry-docs/onboardingdocs/documentationstandards). :

1. Allow users to read through documentation in a clear and concise manner that complies with the [Microsoft Writing Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/).
1. Teams should only add information in their documentation that provides value for the user.

Comment (Rick-Anderson) [reply README.md:69]: [Use bigger ideas, fewer words](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#use-bigger-ideas-fewer-words) : Our modern design hinges on crisp minimalism. Shorter is always better.

**Pull request closed without merging**